src/launch/policy.c

```diff
@@ -886,10 +886,57 @@ static int policy_export_xmit(Policy *policy, CList *list1, CList *list2, sd_bus
                 "a(buu(" POLICY_T_BATCH "))"                                    \
                 "a(ss)"
 
+static int policy_export_console(Policy *policy, sd_bus_message *m, PolicyEntries *entries, uid_t uid_start, uid_t uid_end) {
+        int r;
+
+        if (uid_start >= uid_end)
+                return 0;
```

Comment:
It is really not obvious that uid_end is exclusive. Can we label it as such? uid_end_exclusive? Or better: uid_start and n_uids?

Reply:
Fixed with `n_uids`.